app/views/spree/orders/form/_update_buttons.html.haml

```diff
@@ -1,7 +1,7 @@
 .row
   .columns.small-12.medium-3
-    - if current_order.andand.distributor == @order.distributor
-      - if current_order.line_items.present?
+    - if order.andand.distributor == @order.distributor
+      - if order.line_items.present?
```

Comment:
Are we sure that we don't need that [current_order](https://github.com/coopdevs/spree/blob/spree-upgrade-step-6/core/lib/spree/core/controller_helpers/order.rb#L14)? Because `order` and `@order` are the same thing and that `if` doesn't make any sense anymore.